spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  let(:json_response) { JSON.parse(response.body, symbolize_keys: true) }
+
+  describe 'GET #index' do
+    let!(:organization) { FactoryBot.create(:organization) }
+    let!(:user) { FactoryBot.create(:user, organization: organization) }
+    let(:u_serializer) { UserSerializer }
+
+    context 'Success cases' do
+      def users_from_organization
+        get :index, params: { organization_id: organization.id }
+      end
+
+      before(:each) { users_from_organization }
+
+      it 'should return success' do
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'should return all users from an organization' do
```

Comment:
Maybe it would be great to add a test in which we get more than one user
FactoryBot provides the `create_list` method